pandas/tests/resample/test_datetime_index.py

```diff
@@ -887,13 +922,6 @@ def test_resample_origin_with_tz(unit):
     exp_rng = date_range(
         "1999-12-31 23:57:00", "2000-01-01 01:57", freq="5min", tz=tz
     ).as_unit(unit)
-    resampled = ts.resample("5min", origin="1999-12-31 23:57:00+00:00").mean()
-    tm.assert_index_equal(resampled.index, exp_rng)
```

Comment:
do we have any test left where `origin` is tz-aware? If not, can we restore this one, but with `origin=Timestamp("1999-12-31 23:57:00", tz=tz)`? (something like that, I haven't tested it)

Reply:
good catch